engine/src/test/java/org/camunda/bpm/engine/test/api/mgmt/telemetry/TelemetryReporterTest.java

```diff
@@ -182,7 +181,6 @@ public void tearDown() {
         standaloneProcessEngine.getManagementService().toggleTelemetry(false);
       }
       standaloneProcessEngine.close();
-      ProcessEngines.unregister(standaloneProcessEngine);
```

Comment:
Why don't we need this anymore?

Reply:
ahm, processEngine#close() does that already, I just saw it today

Reply:
OK nice, good catch 👍